lib/llvm/Makefile

```diff
@@ -0,0 +1,64 @@
+# Build llvm library and optionally clang and compiler-rt
```

Comment:
why would we build clang? we are vendoring llvm not the compiler.

Reply:
Vistigual comment, fixing.